lib/rabbitmq/cli/core/helpers.ex

```diff
@@ -187,6 +187,14 @@ defmodule RabbitMQ.CLI.Core.Helpers do
     :net_adm.ping(node) == :pong
   end
 
+  def rabbit_app_running?(%{node: node, timeout: timeout}) do
```

Comment:
But they are predicates. They have to perform I/O so there's potential for other failures. How should they be named if not as predicates?